data.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+// lookup table for buildings that we have in our system.
+var parentNameLookup = map[int]string{
+	146: "Avery",
+	103: "Butler",
+	62:  "East Asian Library",
+	75:  "John Jay",
+	79:  "Lehman Library",
+	84:  "Lerner",
+	15:  "Northwest Corner Building",
+	2:   "Uris",
+}
+
+// dataset is an alias for an array of data dumps
+type dataset []dumpFormat
+
+// dumpFormat represents the datapoints provided for the wireless data.
+// Some fields are gathered from the dumped JSON files and others are configured
+// afterwards before storage.
```

Comment:
Can I assume that the ones with `` `json:"<key>"` `` are denoting the keys as the data is loaded from some JSON file?  Are the other keys gotten dynamically?

My guess:  `DumpTime` is made in this program, `GroupID` is loaded programmatically in `parse_data`, and `ParentName` is generated as we traverse the JSON?

Let me know if I'm wrong about any of that